extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/dataportabilityproject/transfer/smugmug/photos/SmugMugInterface.java

```diff
@@ -16,42 +16,20 @@
 
 package org.dataportabilityproject.transfer.smugmug.photos;
 
-import static com.google.common.base.Preconditions.checkState;
-
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.api.client.http.GenericUrl;
 import com.google.api.client.http.HttpContent;
-import com.google.api.client.http.HttpHeaders;
-import com.google.api.client.http.HttpRequest;
-import com.google.api.client.http.HttpRequestFactory;
-import com.google.api.client.http.HttpResponse;
-import com.google.api.client.http.HttpResponseException;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.http.InputStreamContent;
 import com.google.api.client.http.json.JsonHttpContent;
 import com.google.api.client.json.jackson2.JacksonFactory;
-import com.google.common.base.Charsets;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.io.CharStreams;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Map.Entry;
-import org.dataportabilityproject.transfer.smugmug.photos.model.ImageUploadResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumInfoResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugAlbumsResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugResponse;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugUser;
-import org.dataportabilityproject.transfer.smugmug.photos.model.SmugMugUserResponse;
+import com.google.common.graph.ImmutableGraph;
+import com.google.common.net.HttpHeaders;
+import org.dataportabilityproject.transfer.smugmug.photos.model.*;
```

Comment:
Does google format expand these out or use the wildcard shortcut?

Reply:
I'm not sure - I've seen inconsistencies with running google-format on my mac and on my ubuntu workstation. Other examples are where java standard library imports go (on their own like in this PR, or mixed in with all the other imports.)